MAUI/TreeMap/getting-started.md

```diff
@@ -0,0 +1,515 @@
+            };
+    }
+    public ObservableCollection<PopulationDetails> PopulationDetails
+    {
+        get;
+        set;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+#### Bind data source for TreeMap
+
+To populate the treemap items, utilize the `DataSource` property of `SfTreeMap`. The `PrimaryValuePath` specifies the name of the property within the data object that provides the primary value used to determine the size of each item in the treemap. This primary value typically represents the main quantitative data associated with each item.
+
+{% tabs %}
+{% highlight XAML hl_lines="2 3 4" %}
+<treemap:SfTreeMap x:Name="treeMap"
+                DataSource="{Binding PopulationDetails}"
+                PrimaryValuePath="Population"
+                RangeColorValuePath="Population">
+    <treemap:SfTreeMap.BindingContext>
+        <local:PopulationViewModel />
+    </treemap:SfTreeMap.BindingContext>
+</treemap:SfTreeMap>
+{% endhighlight %}
+{% highlight C# hl_lines="1 3"  %}
+
+SfTreeMap treeMap = new SfTreeMap();
+PopulationViewModel viewModel = new PopulationViewModel();
+treeMap.DataSource = viewModel.PopulationDetails;
+this.Content = treeMap;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Add labels
+
+To display text for the leaf items, utilize the `LabelPath` property within the `LeafItemSettings` of `SfTreeMap`. This property allows you to specify the path to the data property containing the text you want to display.
+
+{% tabs %}
+{% highlight XAML hl_lines="9" %}
+
+<treemap:SfTreeMap x:Name="treeMap"
+                DataSource="{Binding PopulationDetails}"
+                PrimaryValuePath="Population"
+                RangeColorValuePath="Population">
+    <treemap:SfTreeMap.BindingContext>
+        <local:PopulationViewModel />
+    </treemap:SfTreeMap.BindingContext>
+        <treemap:SfTreeMap.LeafItemSettings>
+            <treemap:TreeMapLeafItemSettings LabelPath="Country" 
+                                                Stroke="Black">
+            </treemap:TreeMapLeafItemSettings>
+        </treemap:SfTreeMap.LeafItemSettings>
+</treemap:SfTreeMap>
+
+{% endhighlight %}
+{% highlight C# hl_lines="4"  %}
+
+SfTreeMap treeMap = new SfTreeMap();
+PopulationViewModel viewModel = new PopulationViewModel();
+treeMap.DataSource = viewModel.PopulationDetails;
+this.treeMap.LeafItemSettings = new TreeMapLeafItemSettings() { LabelPath = "Country", Stroke = Brush.Black };
+this.Content = treeMap;
+
+{% endhighlight %}
+{% highlight C# tabtitle="PopulationDetails.cs" %}
+
+// Represents demographic information for a specific country.
+public class PopulationDetails
+{
+    public string Country { get; set; }
+    public string Continent { get; set; }
+    public int Population { get; set; }
+}
+{% endhighlight %}
+{% highlight C# tabtitle="PopulationViewModel.cs" %}
+
+// Presents demographic information for countries in the Americas for the year 2023.
```

Comment:
Represents demographic information for countries in the Americas for the year 2023.

Reply:
Content changed